python/ray/tests/test_structured_logging.py

```diff
@@ -122,6 +122,47 @@ def test_record_with_user_provided_context(self, shutdown_only):
         assert "exc_text" not in record_dict
 
 
+class TestLogfmtFormatter:
```

Comment:
TODO: Add e2e tests for

```python
ray.init(job_config=ray.job_config.JobConfig(py_log_config="TEXT"))
...
```

Reply:
Add two e2e tests https://github.com/ray-project/ray/pull/45344/commits/9b3152ac845d6f7a86ae60f975a035f9481d9409